src/action-sheets/index.js

```diff
@@ -697,7 +697,7 @@ export const constructMessageActionButtons = (args: {|
   if (message.isOutbox === true) {
     buttons.push(copyToClipboard);
     buttons.push(shareMessage);
-    buttons.push(deleteMessage);
+    // buttons.push(deleteMessage);
```

Comment:
Why comment this out? It looks like `deleteMessage` wants to do something interesting when `message.isOutbox === true`.

Reply:
So the idea was to fix issue #5528, and once those changes are merged, I would move on to issue #4701. It states that issue #4701 is blocked on #5528. I only intended to fix issue #5528. 

Your comments make sense. I will leave the button for non-admin user's own messages, and I will replace the if condition to allow users with the minimum role of admin to delete the message. 

I was not sure of whether I should not show the button when message.isOutbox == true. I raised that question in the issue itself. It seems like I should leave it there.

Let me make the proposed changes.

Thanks.